packages/gatsby/lib/query-runner/page-query-runner.js

```diff
@@ -0,0 +1,120 @@
+/***
+ * Jobs of this module
+ * - Ensure on bootstrap that all invalid page queries are run and report
+ *   when this is done
+ * - Watch for when a page's query is invalidated and re-run it.
+ ***/
+
+const _ = require("lodash")
```

Comment:
The general rule I've arrived at (not universally followed) is to use commonjs for node stuff and imports for browser. I write enough "raw" node.js code that getting tripped up by es6 stuff not working when I'm not compiling the code gets really annoying.

For lodash, I just always import the underscore as I dislike going back and forth to change which exact methods I'm importing. When writing browser code, I then just add the webpack/babel lodash plugins to optimize the builds. Which speaking of, it'd be good to get a Gatsby plugin for this.